test/jl/tests.jl

```diff
@@ -142,7 +142,7 @@ function test6()
    println("test 6: Conversion of images from Images.jl to OpenCV Mat")
    filename = joinpath(Pkg.dir("OpenCV"), "./test/images/mandrill.jpg")
    image = imread(filename)
-   dst = Mat(int(rows(image)), int(cols(image)), CV_8UC1)
+   dst = Mat(Int(round(rows(image))), Int(round(cols(image))), CV_8UC1)
    dst = convertToMat(image)
 end
 
```

Comment:
No, I was trying to get over this [error](https://github.com/maxruby/OpenCV.jl/issues/25). I'll test them now.